docs/source/extension/extension_dev.rst

```diff
@@ -214,6 +214,79 @@ See the `JupyterLab Light Theme <https://github.com/jupyterlab/jupyterlab/tree/m
 
 See the `TypeScript extension template <https://github.com/jupyterlab/extension-template>`__ (choosing ``theme`` as ``kind`` ) for a quick start to developing a theme plugin.
 
+Service Manager Plugins
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+   Added in: JupyterLab 4.4.0
```

Comment:
Should we use https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded ?

Reply:
yes good point 👍